include/swift/Basic/TaskQueue.h

```diff
@@ -81,13 +120,15 @@ class TaskQueue {
   /// \param Errors the errors from the task which finished execution, if
   /// available and SeparateErrors was true.  (This may not be available on all
   /// platforms.)
+  /// \param Metrics define basic metrics about the task that ran (has no value
+  /// if no separate process)
```

Comment:
Nitpick: "define" seems like the wrong word here. The parameter isn't defining which metrics are provided, rather it is supplying the values for the metrics.
The parenthetical comment deserves more prominence.
Something like: "If the task ran as a separate process, Metrics supplies the values for the times in took."

Reply:
You're right, I changed the description.